src/instruments/view/instrumentlistmodel.h

```diff
@@ -78,9 +83,36 @@ class InstrumentListModel : public QObject, public async::Asyncable
     void searchStringChanged(QString searchString);
     void selectedInstrumentsChanged();
 
+    void selectedOrderChanged(QString familyId);
+
+    void scoreOrdersChanged();
+
 private:
+    struct SelectedInstrumentInfo
+    {
+        QString id;
+        QString name;
+        QString familyId;
+        bool isSoloist;
```

Comment:
Please, add default values for all bool here (and for ScoreOrderInfo also)

Reply:
Done.